ESLint-Formatter.py

```diff
@@ -7,6 +7,7 @@
 import glob
 import os, sys, subprocess, codecs, webbrowser
 from subprocess import Popen, PIPE
+from SublimeLinter.lint import persist
```

Comment:
SublimeLinter is unrelated to this project and probably doesn't make sense to be a dependency. I previously had assumed that people who used these formatters also had the associated linters installed. That assumption was surprisingly proven wrong.